EIPS/eip-3171.md

```diff
@@ -0,0 +1,148 @@
+---
+eip: 3171
+title: CREATE3
+author: Moody Salem <moody@uniswap.org>
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-12-22
+---
+
+## Simple Summary
+
+This EIP is a fork of [EIP-1014](./eip-1014.md) (`CREATE2`) which adds an account creation opcode (`CREATE3`) that does
```

Comment:
I think this sentence makes more sense in the abstract, and the simple summary can just be sentence below.